### PR TITLE
Feature/banner section

### PR DESCRIPTION
**Pull request by elhaw**
### Description

- added banner html structure and css
- fixed ui bug.

## the current result

### CODE REVIEW

File: src/css/banner.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.banner {
+  font-weight: bold;
```
Comment (tramsi) [inline src/css/banner.css]: Why haven't you used tailwind font weight style ?

**Review by tramsi — DISMISSED** [review]
LGTM!

File: src/css/banner.css (hunk 1 above)
Comment (elhaw) [reply src/css/banner.css]: will use it :+1:

**Review by tramsi — APPROVED** [review]
LGTM

**Pull request merged**